test/com/google/enterprise/adaptor/documentum/DocumentumAdaptorTest.java

```diff
@@ -4597,6 +4597,94 @@ public void testGetGroupsUserMembersOnly() throws Exception {
   }
 
   @Test
+  public void testGetGroups_FullFeeds() throws Exception {
```

Comment:
Surely this is a duplicate, as you suggested? Also, there's only one feed here (not "full feeds" as suggested by the name).

What about actually doing two full feeds, and deleting one of the groups in between them? That's what this feature is actually aimed at, so a test of that seems reasonable.